### PR TITLE
Create initial nuspec and folder structure to build Nuget package

### PR DESCRIPTION
**Pull request by Everest2020**
After discussion in the Teams, we want to create an initial version of nuspec file to try to build Nuget package.

### CODE REVIEW

**Review by jryckman — APPROVED** [review]

File: build/NI.CSharp.Analyzers.nuspec [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <id>NI.CSharp.Analyzers</id>
+    <version>0.1.0</version>
+    <!--TODO: author and owner may have to change if we go to nuget.org -->
+    <authors>NI</authors>
+    <owners>NI</owners>
+    <copyright>Copyright 2020 National Instruments Corporation</copyright>
+    <description>NI's code analyzers and rulesets for .NET projects.</description>
```
Comment (jryckman) [inline build/NI.CSharp.Analyzers.nuspec]: change .NET to C#

File: build/NI.CSharp.Analyzers.nuspec [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <summary>NI's code analyzers and rulesets for .NET projects.</summary>
```
Comment (jryckman) [inline build/NI.CSharp.Analyzers.nuspec]: Change .NET to C#

**Review by jryckman — APPROVED** [review]

File: build/NI.CSharp.Analyzers.nuspec (hunk 1 above)
Comment (Everest2020) [reply build/NI.CSharp.Analyzers.nuspec]: done

File: build/NI.CSharp.Analyzers.nuspec (hunk 2 above)
Comment (Everest2020) [reply build/NI.CSharp.Analyzers.nuspec]: done

File: build/NI.CSharp.Analyzers.nuspec [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <metadata xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <id>NI.CSharp.Analyzers</id>
+    <version>0.1.0</version>
```
Comment (alejandro5042) [inline build/NI.CSharp.Analyzers.nuspec:5]: Would be nice to auto-version this on submit. Something like GitVersion could help (and there are many alternatives!): https://github.com/GitTools/GitVersion

**Pull request merged**